scripts/src/java/org/oppia/android/scripts/maven/GenerateMavenDependenciesList.kt

```diff
@@ -193,6 +259,20 @@ class GenerateMavenDependenciesList(
     }.toSet()
   }
 
+  private fun findFirstDependencyWithBrokenLicense(
+    mavenDependenciesList: List<MavenDependency>,
+    brokenLicenses: Set<License>
+  ): HashMap<License, String> {
```

Comment:
Prefer returning just Map (in general we should use immutable types unless mutability is needed).

Reply:
Now using - 
```
return brokenLicenses.associateTo(mutableMapOf<License, String>()) { license ->
      license to mavenDependenciesList.first { dependency ->
        dependency.licenseList.contains(license)
      }.artifactName
    }
```